### PR TITLE
feat: add jaeger to quick starts

### PR DESCRIPTION
**Pull request by adrielp**

### CODE REVIEW

File: collectors/gateway/colconfig.yaml [hunk 1]
```diff
@@ -54,6 +54,11 @@ exporters:
     tls:
       insecure: true
 
+  otlphttp/jaeger:
+    endpoint: http://jaeger.jaeger-all-in-one-collector.svc.cluster.local:4318
```
Comment (ahmada-liatrio) [inline collectors/gateway/colconfig.yaml:58]: Is this endpoint correct? Can't seem to find where this namespace is created "jaeger-all-in-one-collector"

Comment (adrielp) [reply collectors/gateway/colconfig.yaml]: Fixed

File: collectors/gateway/colconfig.yaml [hunk 2]
```diff
@@ -54,6 +54,11 @@ exporters:
     tls:
       insecure: true
 
+  otlphttp/jaeger:
+    endpoint: http://jaeger-all-in-one-collector.jaeger.svc.cluster.local:4318
```
Comment (ahmada-liatrio) [inline collectors/gateway/colconfig.yaml:58]:
```suggestion
    endpoint: http://jaeger-all-in-one.jaeger.svc.cluster.local:4318
```

Isn't it supposed to be the following based on the deployment file? I don't see where the name appends-collector

Comment (adrielp) [reply collectors/gateway/colconfig.yaml:58]: image

**Review by ahmada-liatrio — APPROVED** [review]
Looking good

**Review by acramsay — APPROVED** [review]

**Pull request merged**